src/main/java/org/quiltmc/loader/impl/metadata/qmj/V1ModMetadataReader.java

```diff
@@ -285,7 +285,8 @@ private V1ModMetadataImpl readFields(JsonLoaderValue.ObjectImpl root) {
 			@Nullable
 			JsonLoaderValue intermediateMappingsValue = quiltLoader.get(QLKeys.INTERMEDIATE_MAPPINGS);
 
-			String[] supported_mappings = { "org.quiltmc:hashed", "net.fabricmc:intermediary" };
+			// TODO: "mojang" for mojmap breaks spec; it is just temporary for now because its experimental
+			String[] supported_mappings = { "org.quiltmc:hashed", "net.fabricmc:intermediary", "mojang" };
```

Comment:
`"net.minecraft:mojang"`?

Reply:
glitch says this is temporary, but well? I know how permanent placeholders can be like!
Anyway, `net.minecraft:mojang` makes sense for dev purposes, but Loom uses `net.minecraft:mappings`, which makes sense but is confusing! (what is Minecraft mappings? I've heard of Mojang!)

So, considering this and Mojang's official standards? Maybe we should go with `com.mojang:mappings`?